src/apis/crypto-compare.ts

```diff
@@ -26,13 +26,27 @@ export async function fetchExchangeRate(
   includeUSDRate?: boolean,
 ): Promise<{
   conversionDate: number;
-  conversionRate: number;
-  usdConversionRate: number;
+  conversionRate: number | null;
+  usdConversionRate: number | null;
 }> {
+  if (!currency || !nativeCurrency) {
+    return {
+      conversionDate: Date.now() / 1000,
+      conversionRate: null,
+      usdConversionRate: null,
+    };
+  }
+
   const json = await handleFetch(
     getPricingURL(currency, nativeCurrency, includeUSDRate),
   );
+
+  if (json.Response === 'Error') {
```

Comment:
Nit: It would be nice to have a comment explaining the CryptoCompare response format (maybe show an example of a failure response?)